### PR TITLE
Resolves Hotfix/Adjust-font-import-urls

### PR DESCRIPTION
**Pull request by Satira**

### CODE REVIEW

File: source/style/modules/sticky-nav.scss [hunk 1]
```diff
@@ -218,7 +218,7 @@ Main Navigation Component
       padding: 3.5vh;
       h4 {
         color: $color-font-primary;
-        font-size: 3vh;
+        font-size: 2.8vh;
```
Comment (chemicalbr) [inline source/style/modules/sticky-nav.scss]: using vh for font size ? Is this ok ?

Comment (Satira) [reply source/style/modules/sticky-nav.scss]: well I guess it's not common practice but it works for me.
Of course I could change it in further commits

Comment (chemicalbr) [reply source/style/modules/sticky-nav.scss]: the question is : Is it good practice, not common.

Comment (Satira) [reply source/style/modules/sticky-nav.scss]: I've added a new issue and will replace all font-size "vh" units with "em/rem" #10

**Pull request merged**